salt/proxy/philips_hue.py

```diff
@@ -294,6 +307,7 @@ def call_status(*args, **kwargs):
     res = dict()
     devices = _get_lights()
     for dev_id in 'id' not in kwargs and sorted(devices.keys()) or _get_devices(kwargs):
+        dev_id = str(dev_id)
```

Comment:
Please replace `str` with `six.text_type` in order to force unicode usage on both Python2 and Python3.

Reply:
Thanks, I've updated this PR accordingly.